svir/dialogs/load_gmf_data_as_layer_dialog.py

```diff
@@ -64,8 +64,40 @@ def __init__(self, iface, viewer_dock, session, hostname, calc_id,
     def set_ok_button(self):
         self.ok_button.setEnabled(self.imt_cbx.currentIndex() != -1)
 
+    def show_num_sites(self):
+        # NOTE: we are assuming all realizations have the same number of sites,
+        #       which is always true for scenario calculations.
+        #       If different realizations have a different number of sites, we
+        #       need to move this block of code inside on_rlz_or_stat_changed()
+        rlz = self.rlz_or_stat_cbx.itemData(
+            self.rlz_or_stat_cbx.currentIndex())
+        gmpe_data = self.npz_file[rlz]
```

Comment:
gmpe_data is the wrong name. If I am not wrong, npz_file contains the GMFs, so the name should be gmf_data